src/date.rs

```diff
@@ -0,0 +1,61 @@
+/// wasm32-unknown-unknown polyfill
+use cfg_if::cfg_if;
+
+cfg_if! {
+    // wasm32-unknown-unknown
+    if #[cfg(target_arch = "wasm32", target_os = "unknown")] {
+        use js_sys::Date;
+        #[derive(Debug, Clone)]
+        pub struct OffsetDateTime(Date);
+        impl OffsetDateTime {
+            #[inline(always)]
+            pub fn now_utc() -> Self {
+                let date = Date::new_0();
+                OffsetDateTime(date)
+            }
+
+            #[inline(always)]
+            pub fn now() -> Self {
+                let date = Date::new_0();
+                OffsetDateTime(date)
+            }
+
+            #[inline(always)]
+            pub fn format(&self, format: impl ToString) -> String {
+                "".into()
```

Comment:
Is this intended to be an empty String? Could you use `date.to_iso_string()` instead?

Reply:
This function seems not easy to implement due to the `format` arg.
Em, I will try to implement one.